app/serializers/banner.ts

```diff
@@ -0,0 +1,25 @@
+import DS from 'ember-data';
+
+import OsfSerializer from './osf-serializer';
+
+export default class BannerAdapter extends OsfSerializer {
+    normalizeSingleResponse(
+        store: DS.Store,
+        primaryModelClass: any,
+        payload: any,
+        id: string,
+        requestType: string,
+    ): any {
+        // The banner payload currently has an empty ID
```

Comment:
Maybe we should "correct" this on the back-end?

Reply:
Sounds worth a ticket, but might require updating the other apps that are already using this API.